common/static/common/js/discussion/views/discussion_content_view.js

```diff
@@ -349,9 +349,9 @@
                 is_subscribing = !this.model.get('subscribed');
                 url = this.model.urlFor(is_subscribing ? 'follow' : 'unfollow');
                 if (is_subscribing) {
-                    msg = gettext('We had some trouble subscribing you to this thread. Please try again.');
+                    msg = gettext('We had some trouble subscribing you to this post. Please try again.');
                 } else {
-                    msg = gettext('We had some trouble unsubscribing you from this thread. Please try again.');
+                    msg = gettext('We had some trouble unsubscribing you from this post. Please try again.');
```

Comment:
"You could not be unsubscribed from this post. Refresh the page and try again."
@alisan617 @andy-armstrong In these cases is there any point to suggesting "Refresh the page"? It sounds better than the curt "Try again", but only if it's an error where refreshing makes sense.